modules/test/playwright/tests/frontend-data-set-admin-web/helpers/DataSetManagerApiHelpers.ts

```diff
@@ -111,19 +111,19 @@ export class DataSetManagerApiHelpers extends ApiHelpers {
 		dataSetERC = DEFAULT_DATA_SET_ERC,
 		icon,
 		label_i18n = {en_US: 'Default Creation Action'},
-		modalSize = 'full-screen',
+		modalSize = EModalActionVariant.FULL_SCREEN,
 		permissionKey,
 		title_i18n,
-		type = 'link',
+		type = ECreationActionType.LINK,
```

Comment:
I prefer `enum` to `type`, as it prevents strings for individual reference, like here. If it was just for variable type, it would be the same.